### DISCUSSION TITLE
Nginx unavailable when I active keycloak socialbackends

### DISCUSSION THREAD

**Discussion by IT-META** [inventree/InvenTree#8568] [Q&A]
Hi everybody,

I have setup a keycloak server on my network for my on premise applications.

I want to use it for Inventree. I have an issue with that. When I configure this :
```
social_backends:
#  - 'allauth.socialaccount.providers.google'
#  - 'allauth.socialaccount.providers.github'
 - 'allauth.socialaccount.providers.keycloak'
```
And restart my server, I have 502 Nginx error.

But when I configured this:
```
social_backends:
  - 'allauth.socialaccount.providers.google'
 - 'allauth.socialaccount.providers.github'
#  - 'allauth.socialaccount.providers.keycloak'
```

The server is starting. I don't understand where is the problem. As someone have this issue ?

Thanks

**Comment by SchrodingersGat** [comment]
Please at bare minimum provide an error log from the InvenTree server.
